assignments/prototypes.js

```diff
@@ -32,79 +51,177 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid(hattrs) {
+  CharacterStats.call(this, hattrs);
+  this.team = hattrs.team;
+  this.weapons = hattrs.weapons;
+  this.language = hattrs.language;
+}
+
+Humanoid.prototype = Object.create(GameObject.prototype);
```

Comment:
We don't need to do this step explicitly, because all of those prototypes are going to by included in our inheritance chain when we get the prototypes from `CharacterStats`